virtualizarr/backend.py

```diff
@@ -133,9 +137,38 @@ def open_virtual_dataset(
         raise NotImplementedError()
 
     # if filetype is user defined, convert to FileType
+
     if filetype is not None:
         filetype = FileType(filetype)
 
+    if filetype == FileType.kerchunk_parquet:
+        from fsspec.implementations.reference import LazyReferenceMapper
+
+        from virtualizarr.readers.kerchunk import dataset_from_kerchunk_refs
+
+        fs = _FsspecFSFromFilepath(filepath=filepath, reader_options=reader_options).fs
+        lrm = LazyReferenceMapper(filepath, fs)
+
+        # build reference dict from KV pairs in LazyReferenceMapper
+        array_refs = {k: lrm[k] for k in lrm.keys()}
+
+        # where does version come from? Is it always 1?
+        full_reference = {"version": 1, "refs": array_refs}
```

Comment:
Isn't the `version` key present in the refs themselves?

Reply:
Good catch. From what I understand** about the Kerchunk spec. `version 1` refers to the current "JSON" structure. The parquet version doesn't have a version number. 

**Could all be very wrong